com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -988,12 +1005,12 @@ void PerformPendingDeletion()
             m_PendingAssetsToBeUnloaded.Clear();
         }
 
-        internal int GetNumberOfBricksAtSubdiv(Cell cell, out Vector3Int minValidLocalIdxAtMaxRes, out Vector3Int sizeOfValidIndicesAtMaxRes)
+        internal int GetNumberOfBricksAtSubdiv(Vector3Int position, int minSubdiv, out Vector3Int minValidLocalIdxAtMaxRes, out Vector3Int sizeOfValidIndicesAtMaxRes)
```

Comment:
Any reason for not passing the cell here and split the two fields? The cell gives us more info at disposal in case we'd need to change this logic. 

Tbf, don't care much either way, but I wonder if you had a strong reason to

Reply:
Because it is invoked during baking in a location that now only has access to BakingCell before any Cell has been created.